packages/react-native-renderer/src/ReactNativeFiberInspector.js

```diff
@@ -117,13 +147,93 @@ if (__DEV__) {
       source,
     };
   };
+
+  getInspectorDataForViewAtPoint = function(
+    findNodeHandle: (componentOrHandle: any) => ?number,
+    inspectedView: Object,
+    locationX: number,
+    locationY: number,
+    callback: (viewData: TouchedViewDataAtPoint) => mixed,
+  ): void {
+    let closestInstance = null;
+    let frame = {
```

Comment:
Do you still need to define this here since you define it inline in the callback?

Reply:
Yes because of the early return but I moved it inline for the early return.